src/DataProtection/docs/Set-AzDataProtectionMSIPermission.md

```diff
@@ -0,0 +1,210 @@
+---
+external help file:
+Module Name: Az.DataProtection
+online version: https://docs.microsoft.com/powershell/module/az.dataprotection/set-azdataprotectionmsipermission
+schema: 2.0.0
+---
+
+# Set-AzDataProtectionMSIPermission
+
+## SYNOPSIS
+Grants required permissions to the backup vault to configure backup
+
+## SYNTAX
+
+```
+Set-AzDataProtectionMSIPermission -BackupInstance <IBackupInstanceResource> -PermissionsScope <String>
+ -VaultName <String> -VaultResourceGroup <String> [-KeyVaultId <String>] [-Confirm] [-WhatIf]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+Grants required permissions to the backup vault to configure backup
+
+## EXAMPLES
+
+### Example 1: Grant Permissions for Azure Disks
+```powershell
+PS C:\> Set-AzDataProtectionMSIPermission -BackupInstance $instance -VaultResourceGroup "VaultRG"" -VaultName "Vaultname"" -PermissionsScope "ResourceGroup"
```

Comment:
Please remove PS C:\> from all examples to avoid breaking user's copy-paste-use experience.